engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglGraphics.java

```diff
@@ -254,6 +254,7 @@ private void checkOpenGL() {
                 GLContext.getCapabilities().OpenGL12,
                 GLContext.getCapabilities().OpenGL14,
                 GLContext.getCapabilities().OpenGL15,
+                GLContext.getCapabilities().OpenGL20,
```

Comment:
Well, by shader format I pretty much just mean a way to pack multiple versions of the shader in one file (and probably both the vert and frag shaders) in the first instance. 

I don't think it would be to hard to go further though, if need be.